installation-and-configuration/configure-winrm-authentication.md

```diff
@@ -0,0 +1,33 @@
+---
+description: >-
+  This article explains how to configure WinRm authentication on
+  your machine to successfully run snapshots.
+---
+
+# Configure Windows Remote Management (WinRM)
+
+In order to successfully collect data when creating a snapshot, **SysKit Trace** will use a **remote PowerShell session** to connect to the Office 365. This is required to collect the data for Security & Compliance, Exchange Online and some Teams reports.
```

Comment:
i think it's usually used without **the**. to connect to Office 365 sounds more correct.